apps/trivium/src/kreyvium/kreyvium_shortint.rs

```diff
@@ -0,0 +1,206 @@
+use crate::static_deque::StaticDeque;
+
+// use tfhe::prelude::*;
```

Comment:
check for comments like this to be removed in other files, I think I saw it in a few places